package.json

```diff
@@ -14,8 +14,9 @@
     "lint:styles": "stylelint \"src/**/*.{css,sass,scss}\" --max-warnings=0",
     "lint:scripts": "eslint \"{src,config}/**/*.{js,ts,jsx,tsx}\" --cache --fix --max-warnings=0 && tsc --noEmit",
     "preview": "vite preview",
-    "rsc": "node scripts/release-sanity-check.mjs",
-    "generate-ether-types": "typechain --target=ethers-v5 'abis/**/*.json' --out-dir src/types/contracts"
+    "generate-ether-types": "typechain --target=ethers-v5 'abis/**/*.json' --out-dir src/types/contracts",
+    "package-version": "node -p \"require('./package.json').version\"",
+    "version": "rlx changelog release $(yarn package-version) && git add ."
```

Comment:
Would be nice to put it into the `preversion` hook but I guess it won't fit into the current CI workflow

Reply:
@ihordiachenko the package.json version is not bumped yet when the `preversion` hook is called. And you still need to stage changed files using 'git add`: https://docs.npmjs.com/cli/v9/commands/npm-version#description

Reply:
Yeah, you're right. Re-checked the shelf stuff and it also uses the version hook